diet-error-handler/Gemfile.lock

```diff
@@ -1,29 +1,29 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    backports (3.11.0)
+    backports (2.8.2)
     json (2.1.0)
     mini_portile2 (2.3.0)
-    multi_json (1.13.0)
-    mustermann (1.0.1)
+    multi_json (1.13.1)
+    mustermann (1.0.3)
     nokogiri (1.8.5)
       mini_portile2 (~> 2.3.0)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.7)
+    rack-protection (2.0.2)
       rack
-    sinatra (2.0.0)
+    sinatra (2.0.2)
```

Comment:
Your PR says 2.0.0 to 2.0.5 but this is 2.0.2.  Is this intended?

Reply:
Whoops - I thought I'd updated the PR title. It should be 2.0.2 - I'll update now. Well spotted.

Reply:
(I did try 2.0.5 initially but that broke the build).